frontend/src/layouts/page/page.tsx

```
@@ -3,34 +1,107 @@
+
+      {errors?.map((error, i) => (
+        <Alert variant={error.variant as AlertProps['variant']} key={i} className="page-error">
+          {error.text}
+        </Alert>
+      ))}
+
+      {!hasRights && (
+        <Alert variant="danger">
+          <Trans
+            i18nKey="page.access-denied"
+            components={[<a href="/#!/access-level" key="access-level" />]}
```

Biome (1.9.4)

[error] Provide screen reader accessible content when using `a` elements.

All links on a page should have content that is accessible to screen readers.
Accessible content refers to digital content that is designed and structured in a way that makes it easy for people with disabilities to access, understand, and interact with using assistive technologies.
Follow these links for more information,
 WCAG 2.4.4
 WCAG 4.1.2

(lint/a11y/useAnchorContent)